routing/pathfind.go

```diff
@@ -596,6 +630,49 @@ func findPath(tx *bolt.Tx, graph *channeldb.ChannelGraph,
 		if err != nil {
 			return nil, err
 		}
+
+		// Then, we'll examine all the additional edges from the node
+		// we're currently visiting.
+		for _, edge := range additionalEdges[bestNode.PubKeyBytes] {
+			v := Vertex(edge.Node.PubKeyBytes)
```

Comment:
Since you are doing the same here and in the loop above, this can be extracted into a helper closure that can be called from both loops:

```go
for _, edge := range additionalEdges[bestNode.PubKeyBytes] {
    capacity := unknownCapacity
    processEdge(edge, capacity)
```

Reply:
Fixed.